src/main/java/org/jitsi/videobridge/RtpChannel.java

```diff
@@ -1801,17 +1761,18 @@ public boolean expire()
                 conferenceStatistics.totalNoPayloadChannels.incrementAndGet();
             }
 
+            MediaStreamStats streamStats = stream.getMediaStreamStats();
```

Comment:
Good point, but we do assume the existence of MediaStreamStats in various places and the MediaStream JavaDoc doesn't include null as a valid return value.